deploy/osd-splunk-audit-exporter/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
I think this `osd-splunk-audit-exporter` directory should be move out of /deploy, because it will be deployed to the hive cluster itself. (see below)

Reply:
Hm I tried to move it out - but obviously then it won't generate the policy file anymore (even if I try to put in a relative path like `../osd-splunk-audit-exporter`). But if I try to make the python file pick up that path again it will also include it in the regular OSD clusters again.